homeassistant/components/verisure/__init__.py

```diff
@@ -108,7 +108,7 @@ async def async_migrate_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
             hass.config_entries.async_update_entry(entry, options=new_options)
 
-        entry.version = 2
+        hass.config_entries.async_update_entry(entry, version=2)
```

Comment:
I hate to call this twice, but I didn't want to refactor and introduce risk since I can't test this as I don't have this integration